mkosi

```diff
@@ -427,16 +434,21 @@ def install_fedora(args, workspace, run_build_script):
     if os.path.exists(gpg_key):
         gpg_key = "file://%s" % gpg_key
     else:
-        gpg_key = "https://getfedora.org/static/81B46521.txt"
+        gpg_key = "https://getfedora.org/static/{key}.txt".format(key=FEDORA_KEYS_MAP[args.release])
 
     if args.mirror:
-        release_url = "baseurl={.mirror}/releases/{.release}/Everything/x86_64/os/".format(args)
-        updates_url = "baseurl={.mirror}/updates/{.release}/x86_64/".format(args)
+        if args.release == "25":
+            section = "development"
+        else:
+            section = "releases"
+
+        release_url = "baseurl={args.mirror}/{section}/{args.release}/Everything/x86_64/os/".format(args=args, section=section)
+        updates_url = "baseurl={args.mirror}/updates/{args.release}/x86_64/".format(args=args)
```

Comment:
I couldn't find a definitive place for this, so I guess we'll just have to try `releases/` first and fallback to `development/` if that fails, probably checking for something like `$section/$release/Everything/$arch/os/media.repo`.